### PR TITLE
feat(core): auto sign-out

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

auto sign-out for rp initiated logout.

ref: https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#logoutsource

## Testing

- [x] localhost sign out without manual action

### CODE REVIEW

File: packages/core/static/html/logout.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#logoutsource -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (demonzoo) [inline packages/core/static/html/logout.html:6]: This can be removed, since IE is finally gone

File: packages/core/static/html/logout.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (demonzoo) [inline packages/core/static/html/logout.html:7]: Need to disable pinch zooming here, as it might be integrated into webview on mobile

Comment (gao-sun) [reply packages/core/static/html/logout.html:7]: this page will not be able to interact, just a loading indicator

Comment (demonzoo) [reply packages/core/static/html/logout.html:7]: Ah, right... hmm, then that's ok

**Review by demonzoo — APPROVED** [review]
Looks good

File: packages/core/src/middleware/koa-spa-proxy.ts [hunk 3]
```diff
@@ -16,7 +15,7 @@ export default function koaSpaProxy<StateT, ContextT extends IRouterParamContext
 ): MiddlewareType<StateT, ContextT, ResponseBodyT> {
   type Middleware = MiddlewareType<StateT, ContextT, ResponseBodyT>;
 
-  const packagesPath = fromRoot ? 'packages/' : '..';
+  const packagesPath = '..';
   const distPath = path.join(packagesPath, packagePath, 'dist');
```
Comment (IceHe) [inline packages/core/src/middleware/koa-spa-proxy.ts:19]: nit: maybe better inline?

**Review by IceHe — APPROVED** [review]
Overall LGTM

**Pull request merged**